.github/workflows/pod_failover_test.yml

```diff
@@ -0,0 +1,81 @@
+name: Pod Failover Test
+on:
+    schedule:
+    - cron: "0 13 * * *"
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      AZURE_TENANT_ID: ${{ secrets.TenantId }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.SubscriptionId }}
+      AZURE_CLIENT_ID: ${{ secrets.ClientId }}
+      AZURE_CLIENT_SECRET: ${{ secrets.ClientSecret }}
+      AZURE_LOCATION: ${{ secrets.E2ELocation }}
+      REGISTRY: e2etest.azurecr.io/mandalay
+      METRICS_ENDPOINT: ${{ secrets.MetricsEndpoint }}
+    
+    steps:
+    - name: Set UUID
+      id: generate-uuid
+      uses: filipstefansson/uuid-action@v1
+
+    - name: Set Azure resource group
+      run: |
+        echo "AZURE_RESOURCE_GROUP=k8s-e2eTest-${{ steps.generate-uuid.outputs.uuid }}-rg" >> $GITHUB_ENV
+ 
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.16
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install conntrack
+      run: sudo apt install conntrack
+
+    - name: Login to Azure
+      run: |
+        az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID
+
+    - name: Login to aks registry
+      run: az acr login --name e2etest
+
+    - name: Build Test
+      env:
+        BUILD_V2: true
+      run: |
+        make container-all push-manifest
+
+    - name: Create an aks cluster 
+      run: |  
+        ./hack/azure-cluster-up.sh --subscription $AZURE_SUBSCRIPTION_ID --location $AZURE_LOCATION --output e2eTestOutput --client-id $AZURE_CLIENT_ID --client-tenant $AZURE_TENANT_ID --client-secret $AZURE_CLIENT_SECRET --resource-group $AZURE_RESOURCE_GROUP
+        echo "KUBECONFIG=$GITHUB_WORKSPACE/e2eTestOutput/kubeconfig/kubeconfig.${AZURE_LOCATION}.json" >> $GITHUB_ENV
+
+    - name: Deploy the driver on the cluster
+      env:
+        BUILD_V2: true
+      run: make e2e-bootstrap
+
+    - name: Build controller and workload containers
+      run: make pod-failover-test-containers
```

Comment:
Currently, the failover container images are push to constant names. Consider suffixing them with the Git commit if the `CI` environment variable is defined like with do for the driver container images. (See https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/e9c81c18de3a5886020b84f82d88ed8c4f335efa/Makefile#L35.)

Reply:
Done